### ISSUE TITLE
Request: Add a column to the CSV file showing the last stage that successfully completed

### ISSUE THREAD

**Issue by jeremyfowers** [onnx/turnkeyml#23]
# Issue

Add a column to the CSV file showing the last stage that successfully completed

This is needed because the "completed stages" with their respective times is very hard to parse on excel.

 @jeremyfowers @ramkrishna2910 
 
 Requested by @amacias

**Comment by danielholanda** [comment]
@jeremyfowers Can we close this now that we have `stage_status` as part of the report?
